tests/integration/utils.ts

```diff
@@ -73,29 +73,36 @@ export async function extensionLogsPresent(
 ): Promise<boolean> {
```

Comment:
just noticed, the function param description above (line 64) should be updated for the new array of RegExp matches instead of a single message